app/controllers/analytics_controller.rb

```diff
@@ -0,0 +1,22 @@
+class AnalyticsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :confirm_two_factor_authenticated
+
+  def create
+    unless analytics_saved?
+      session[:platform_authenticator] = true
+      analytics.track_event(Analytics::PLATFORM_AUTHENTICATOR, results.to_h)
+    end
+    head :ok
+  end
+
+  private
+
+  def results
+    FormResponse.new(success: params[:available], errors: {})
```

Comment:
This allows someone to put arbitrary content into our analytics database. We probably have similar issues elsewhere, but here, availability should be boolean.

Reply:
Nothing prevents them from hitting us ad infinitum either.  My thoughts were if  isUserVerifyingPlatformAuthenticatorAvailable decides to send back a value other than true or false we'd like to know it.